pkg/grpc/exec/exec.go

```diff
@@ -306,10 +315,9 @@ func GetProcessExit(event *MsgExitEventUnix) *tetragon.ProcessExit {
 		parent.RefDec()
 	}
 	if proc != nil {
-		tetragonEvent.Process = proc.GetProcessCopy()
+		// Use cached version of process
+		tetragonEvent.Process = proc.UnsafeGetProcess()
 		proc.RefDec()
```

Comment:
so the changelog says you want to ensure PID == TID for exec/exit events,
but the change has also changes above that apparently do something else,
could we split that into separate commit with its own explanation?

Reply:
You are completely right! I did split them and re-wrote extensive commit logs for it.

Sorry for that.